man/systemd.exec.xml

```diff
@@ -200,6 +200,19 @@
         <xi:include href="system-only.xml" xpointer="singular"/></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>RootEphemeral=</varname></term>
+
+        <listitem><para>Takes a boolean argument. If enabled, executed processes will run in an ephemeral
+        copy of the root directory or root image. The ephemeral copy is placed in
+        <filename>/var/lib/systemd/ephemeral/</filename> while the service is active and is cleaned up when
+        the service is stopped or restarted. If <varname>RootDirectory=</varname> is used and the root
+        directory is a subvolume, the ephemeral copy will be created by making a snapshot of the subvolume.
+        </para>
+
```

Comment:
so this is only going to be efficient on btrfs systems (and to some degree on reflink-based systems). Moreover, to be efficient the root fs must be on the same fs as /var/lib/systemd/ephemeral/.

I think this deserves some brief discussion here

Reply:
of course, for disk images reflink bsaed is fully efficient. for dir trees subvols are much more efficient.